fan-c3.yaml

```
@@ -0,0 +1,47 @@
+esphome:
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)